addons/resource.language.en_gb/resources/strings.po

```diff
@@ -19464,7 +19464,31 @@ msgctxt "#38042"
 msgid "[Missing]"
 msgstr ""
 
-#empty strings from id 38043 to 38099
+#. Title album artists node
+#: system/library/music/musicroles/albumartists.xml
+msgctxt "#38043"
+msgid "Album Artists"
+msgstr ""
+
+#. Title all artists node
+#: system/library/music/musicroles/allartists.xml
+msgctxt "#38044"
+msgid "Song & Album Artists"
+msgstr ""
+
+#. Title all contributors node (any role)
+#: system/library/music/musicroles/allcontributors.xml
+msgctxt "#38045"
+msgid "All Contributors"
```

Comment:
Since other nodes are only first word capitals e.g. "Recently played albums", I have adjusted strings.po to match. Thanks for picking this up.

(oddly new GitHub keeps showing the now out of date line above and expanded comments)